inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -47,10 +47,13 @@ void LayerTestsCommon::Run() {
 
```

Comment:
Looks like, we should update the code related to reporting:
`s.updateOPsStats(functionRefs, ...`

Reply:
Update codes as you commented.